libraries/wavemap_io/src/stream_conversions.cc

```diff
@@ -115,6 +116,7 @@ bool streamToMap(std::istream& istream, WaveletOctree::Ptr& map) {
   std::stack<WaveletOctree::NodeType*> stack;
   stack.emplace(&map->getRootNode());
   while (!stack.empty()) {
+    DCHECK(!stack.empty());
```

Comment:
How could this check ever fail?

Reply:
Also a leftover I must have overlooked when refactoring. Removed it. Thanks for pointing it out!